ray-operator/controllers/ray/raycluster_controller_fake_test.go

```diff
@@ -2340,6 +2343,171 @@ func TestReconcile_Replicas_Optional(t *testing.T) {
 	}
 }
 
+func TestReconcile_Multihost_Replicas(t *testing.T) {
+	setupTest(t)
+
+	// This test makes some assumptions about the testRayCluster object.
+	// (1) 1 workerGroup (2) disable autoscaling
+	assert.Equal(t, 1, len(testRayCluster.Spec.WorkerGroupSpecs), "This test assumes only one worker group.")
+
+	// Disable autoscaling so that the random Pod deletion is enabled.
```

Comment:
Random pod deletion is a pretty bad behavior for a multi-host setup. I am considering disabling it.